<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>971</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="mainWidget" native="true">
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>941</width>
          <height>531</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="mainLayout" stretch="1,16">
         <property name="sizeConstraint">
          <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QWidget" name="titleBar" native="true">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>1111</width>
             <height>1111</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <widget class="QWidget" name="horizontalLayoutWidget">
            <property name="geometry">
             <rect>
              <x>-1</x>
              <y>-1</y>
              <width>941</width>
              <height>31</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,19,1,1,1">
             <property name="sizeConstraint">
              <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
             </property>
             <item>
              <widget class="QLabel" name="APPpixLable">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="appnamelabel">
               <property name="font">
                <font>
                 <family>HarmonyOS Sans SC</family>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>PhotoShop-Lite</string>
               </property>
               <property name="textFormat">
                <enum>Qt::TextFormat::RichText</enum>
               </property>
               <property name="margin">
                <number>2</number>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="minBtn">
               <property name="text">
                <string/>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="maxBtn">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="closeBtn">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="Displaywidget" native="true">
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>70</y>
              <width>161</width>
              <height>351</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="mode1Btn">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>136</width>
                 <height>48</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="mode2Btn">
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>136</width>
                 <height>48</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="mode3Btn">
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>136</width>
                 <height>48</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="mode4Btn">
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>136</width>
                 <height>48</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QTextEdit" name="Pic_filepath_textEdit">
            <property name="geometry">
             <rect>
              <x>230</x>
              <y>410</y>
              <width>701</width>
              <height>41</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="filepath_label">
            <property name="geometry">
             <rect>
              <x>240</x>
              <y>420</y>
              <width>701</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
           <widget class="QLabel" name="Pic_label">
            <property name="geometry">
             <rect>
              <x>230</x>
              <y>40</y>
              <width>670</width>
              <height>340</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="backgroundlabel">
            <property name="geometry">
             <rect>
              <x>220</x>
              <y>30</y>
              <width>690</width>
              <height>360</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <zorder>backgroundlabel</zorder>
           <zorder>layoutWidget</zorder>
           <zorder>Pic_filepath_textEdit</zorder>
           <zorder>filepath_label</zorder>
           <zorder>Pic_label</zorder>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
